spec/integ/admin-rooms.spec.js

```diff
@@ -885,14 +885,12 @@ describe("Admin rooms", function() {
                     cmdIx++;
                 });
 
-            // 5 commands should be executed
-            // rubbishserver should not be accepted
+            // 4 commands should be executed
             const commands = [
                 `!cmd ${roomMapping.server} JOIN ${newChannel}`,
                 `!cmd ${roomMapping.server} TOPIC ${newChannel} :some new fancy topic`,
                 `!cmd ${roomMapping.server} PART ${newChannel}`,
-                `!cmd ${roomMapping.server} STUPID COMMANDS`,
-                `!cmd rubbishserver SOME COMMAND`];
```

Comment:
we've lost the test for an invalid server here?

Reply:
With this change "rubbishserver" is now interpreted as a command rather than as a server.

Reply:
right...we still need to interpret servers here.

Reply:
They are interpreted correctly when valid, but `rubbishserver` matches the new regex.

Perhaps that could be replaced by `rubbish^server!` or something?